mitxpro/tests/render_bundle_test.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 
 from mitxpro.utils import webpack_dev_server_url
-from mitxpro.templatetags.render_bundle import render_bundle
```

Comment:
Since you're touching these lines, could you just removed `render_bundle` and its tests entirely? `mitol-django-common` should be providing them now.